cmake/externals/sentencepieceproject.cmake

```diff
@@ -23,29 +66,22 @@ FetchContent_Declare(
 )
 FetchContent_GetProperties(spm)
 
-if(_ONNXRUNTIME_EMBEDDED)
-  set(SPM_USE_BUILTIN_PROTOBUF OFF CACHE INTERNAL "")
+set(SPM_USE_EXTERNAL_ABSL OFF CACHE BOOL "Use external absl" FORCE)
```

Comment:
If this is an embedded build should we be pointing it to the absl library ORT is using (unless onnxruntime_DISABLE_ABSEIL is on)?

Reply:
this flag is misleading, and it doesn't work as what we expected.
https://github.com/google/sentencepiece/blob/c5a49eb5de2867f4f31aca0abc1c70c35605a5f6/CMakeLists.txt#L142

Actually, to make abseil external library work, it requires another big patch on SPM library and may be do it later.